### PR TITLE
Render locked item/fluid for empty super/quantum containers

### PR DESCRIPTION
**Pull request by mikerooni**
image

### CODE REVIEW

File: common/src/main/java/com/gregtechceu/gtceu/common/machine/storage/QuantumChestMachine.java [hunk 1]
```diff
@@ -74,7 +77,7 @@ public class QuantumChestMachine extends TieredMachine implements IAutoOutputIte
     protected ISubscription exportItemSubs;
     @Persisted @Getter @Setter
     private boolean isVoiding;
-    @Persisted @Getter
+    @Persisted @DescSynced @Getter
```
Comment (screret) [inline common/src/main/java/com/gregtechceu/gtceu/common/machine/storage/QuantumChestMachine.java:80]: Should this maybe also have `@RequireRenderer`? Unsure

File: common/src/main/java/com/gregtechceu/gtceu/common/machine/storage/QuantumTankMachine.java [hunk 2]
```diff
@@ -74,7 +74,7 @@ public class QuantumTankMachine extends TieredMachine implements IAutoOutputFlui
     protected FluidStack stored = FluidStack.empty();
     @Persisted @Getter @Setter
     private boolean isVoiding;
-    @Persisted @Getter
+    @Persisted @DescSynced @Getter
```
Comment (screret) [inline common/src/main/java/com/gregtechceu/gtceu/common/machine/storage/QuantumTankMachine.java:77]: Same for this

File: common/src/main/java/com/gregtechceu/gtceu/common/machine/storage/QuantumChestMachine.java (hunk 1 above)
Comment (mikerooni) [reply common/src/main/java/com/gregtechceu/gtceu/common/machine/storage/QuantumChestMachine.java:80]: The renderer runs on every client tick (or frame? unsure) anyway, so I've decided to use the same annotations we had on `stored` for this as well.

Comment (screret) [reply common/src/main/java/com/gregtechceu/gtceu/common/machine/storage/QuantumChestMachine.java:80]: Ok cool

**Review by screret — APPROVED** [review]

**Pull request merged**